packages/lodestar/src/chain/chain.ts

```diff
@@ -133,9 +135,34 @@ export class BeaconChain extends (EventEmitter as { new(): ChainEventEmitter })
     } catch (e) {
       return;
     }
+    const targetRoot = attestation.data.target.root;
+    if (!await this.db.block.has(targetRoot)) {
+      this.emit("unknownBlockRoot", targetRoot);
+      this.attesationWaitingQueue.add(attestation);
+      return;
+    }
+    const beaconBlockRoot = attestation.data.beaconBlockRoot;
+    if (!await this.db.block.has(beaconBlockRoot)) {
+      this.emit("unknownBlockRoot", beaconBlockRoot);
+      this.attesationWaitingQueue.add(attestation);
+      return;
+    }
```

Comment:
You could squash those if statements into single one

Reply:
I can't because it's not doing the same thing in the body of these `if` statements.